flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -121,11 +125,10 @@
 				.list(
 					text("'full': Restarts all tasks to recover the job."),
 					text("'region': Restarts all tasks that could be affected by the task failure. " +
-						"The affected vertices consist of vertices in affected regions " +
-						"(pipelined connected sub-graphs), including " +
-						"the region of the failed task, " +
-						"downstream regions of affected regions, and" +
-						"producer regions of required but unavailable result partitions for this recovery.")
+						"More details can be found %s.",
+						link(
+							"https://ci.apache.org/projects/flink/flink-docs-master/dev/task_failure_recovery.html#restart-pipelined-region-failover-strategy",
```

Comment:
At the moment there is no possibility to reference the site root from here. I would avoid linking the *master* docs here. The best alternative I can think of is to use a relative path: `../dev/task_failure_recovery.html#restart-pipelined-region-failover-strategy`. However, this assumes a specific directory structure.

Reply:
Ok. We can use the relative path. No other good idea yet.